test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -154,9 +154,14 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 				klog.InfoS("computed base load", "value", baseload)
 				paddingResWithBaseload := paddingRes.DeepCopy()
 				baseload.Apply(paddingResWithBaseload)
+				var zonePaddingRes corev1.ResourceList
 				for zIdx, zone := range nrtInfo.Zones {
+					zonePaddingRes = paddingRes
+					if zIdx == 0 {
+						zonePaddingRes = paddingResWithBaseload
+					}
 					podName := fmt.Sprintf("padding-%d-%d", nIdx, zIdx)
-					padPod, err := makePaddingPod(fxt.Namespace.Name, podName, zone, paddingResWithBaseload)
+					padPod, err := makePaddingPod(fxt.Namespace.Name, podName, zone, zonePaddingRes)
```

Comment:
there is no correct way to account for baseload per-NUMA because the baseload is composed by burstable and best-effort pods (most notably burstable) which can move freely around NUMA zones. Accounting against any NUMA zone is never fully correct.

This means that every possible solution is unavoidably partial and has pros and cons.